python/cudf/cudf/tests/test_datetime.py

```diff
@@ -1297,3 +1297,27 @@ def test_is_leap_year():
     got2 = gIndex.is_leap_year
 
     assert_eq(expect2, got2)
+
+
+def test_is_month_start():
+    data = [
+        "2020-05-31",
+        None,
+        "1999-12-01",
+        "2000-12-21",
+        None,
+        "1900-02-28",
+        "1800-03-14",
+        "2100-03-10",
+        "1970-01-01",
+        "1969-12-11",
+    ]
+
+    # Series
+    ps = pd.Series(data, dtype="datetime64[ms]")
```

Comment:
this does not affect the actual results, but pandas only supports `datetime64[ns]` and is autoconverting this.